aries-site/src/pages/develop/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+
 import { PageLayout } from '../../layouts';
-import { ComingSoon, DescriptiveHeader } from '../../components';
-import { structure } from '../../data';
+import { ComingSoon, DescriptiveHeader, Meta } from '../../components';
+import { getPageDetails } from '../../utils';
 
 const title = 'Develop';
-const topic = structure.find(page => page.name === title);
+const topic = getPageDetails(title);
```

Comment:
Wondering if we should change this to be `const page = getPageDetails(title);`to be consistent with other pages since the page title is what we are calling on topic on the other pages. Seems a little confusing to be mixing terms.

Reply:
I can get onboard with that. Will do for each of the Routing Pages.

Reply:
Updated per your suggestion.